docs/current/guides/snippets/use-services/expose-host-services-to-container/main.go

```diff
@@ -0,0 +1,39 @@
+package main
```

Comment:
@helderco @TomChv @slumbering could you help me with equivalent code in Typescript and Python?

Reply:
@slumbering make sure you wrap the command in ./hack/dev so it runs against the dev engine

Reply:
Thanks it works indeed 🙏